Leanplum-SDK/Classes/Notifications/LPNotificationsHelper.h

```diff
@@ -0,0 +1,20 @@
+//
+//  LPNotificationsHelper.h
+//  Leanplum-iOS-SDK
+//
+//  Created by Dejan . Krstevski on 15.05.20.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface LPNotificationsHelper : NSObject
```

Comment:
how are the helper vs manager responsibilities split up?

Reply:
was planning to do something like common manager for local and push.... I named it helper (not the best name), planning to refactor this